pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -144,6 +144,10 @@ func newCertRotationController(
 			Lister:                 kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets().Lister(),
 			Client:                 kubeClient.CoreV1(),
 			EventRecorder:          eventRecorder,
+
+			// we will remove this when we migrate all of the affected secret
+			// objects to their intended type: https://issues.redhat.com/browse/API-1800
+			UseSecretUpdateOnly: true,
```

Comment:
we should also apply it to the `RotatedSelfSignedCertKeySecret` controller, right ?